### DISCUSSION TITLE
为什么一段长文本会出现多种不同的声音

### DISCUSSION THREAD

**Discussion by yincangshiwei** [lenML/Speech-AI-Forge#37] [问答 | Q&A]
我的一段长文本，会出现不同人对话的场景，不知哪里配置有问题。
配置如下：
![image](https://github.com/lenML/ChatTTS-Forge/assets/8185843/9724555b-c055-40c6-bb64-327cf3a1273b)

**Comment by zhzLuke96** [comment]
命令行有没有报错或者warning？

没有的话试试把温度拉到最低能不能固定

要是还是不能固定的话提供更详细的环境状态：
比如启动之后页面最下面的那一栏信息
以及你的操作系统

**Comment by yincangshiwei** [comment]
> 命令行有没有报错或者warning？
> 
> 没有的话试试把温度拉到最低能不能固定
> 
> 要是还是不能固定的话提供更详细的环境状态： 比如启动之后页面最下面的那一栏信息 以及你的操作系统

我试了官方的原版，也是有这问题，我试下调温度看看。
